### PR TITLE
mypy: update to 0.780

### PR DESCRIPTION
**Pull request by singingwolfboy**
http://mypy-lang.blogspot.com/2020/06/mypy-0780-released.html

### CODE REVIEW

File: Formula/mypy.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/python/mypy.git",
-      :tag      => "v0.770",
-      :revision => "92e3f396b03c69c1d8abc249632bfd2db02350d0"
+      :tag      => "v0.780",
+      :revision => "3423b18a30ac49d6e7050abeb188ad3ba5d23d35"
   revision 1
```
Comment (bayandin) [inline Formula/mypy.rb:7]: This line should be removed

Comment (singingwolfboy) [reply Formula/mypy.rb]: Thanks! Fixed.

**Comment by bayandin** [comment]
And could you please squash commits into one `mypy 0.780`, and we're ready to go 🎉

**Review by bayandin — APPROVED** [review]
Thanks @singingwolfboy!

**Pull request closed without merging**